jitpack.yml

```diff
@@ -1,5 +1,4 @@
-before_install:
-  - wget https://github.com/sormuras/bach/raw/releases/11/install-jdk.sh
-  - source install-jdk.sh --feature 17
+jdk:
+  - openjdk17
```

Comment:
This works? :O Did JitPack introduce this recently? :O

Reply:
Not sure when it became a thing, but I've been using it for a while now. You can also use SDKMan if you need a specific JDK version